item/serializers.py

```diff
@@ -0,0 +1,95 @@
+from rest_framework import serializers
+from item.validators import positive_price_validator
+from item.models import Currency, Item, Price
+
+
+class CurrencySerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Currency
+        fields = ('code', 'name')
+        read_only_field = ('id',)
+
+    code = serializers.CharField(max_length=8)
+    name = serializers.CharField(max_length=128)
```

Comment:
in some places you need to specify Field directly but it's for `SlugRelatedField` etc.
Here there is no reason to specify them